javalib/src/main/scala/java/lang/impl/PosixThread.scala

```diff
@@ -263,10 +261,9 @@ private[java] class PosixThread(val thread: Thread, stackSize: Long)
         fromRawPtr(elemRawPtr(base, offset))
     }
 
-// TODO: atomics
-//  @alwaysinline private def counterAtomic = new CAtomicInt(
-//    fromRawPtr(classFieldRawPtr(this, "counter"))
-//  )
+  @alwaysinline private def counterAtromic = new CAtomicInt(
```

Comment:
not sure how many references there are to this definition
```suggestion
  @alwaysinline private def counterAtomic = new CAtomicInt(
```